src/rewriter.coffee

```diff
@@ -267,7 +267,8 @@ exports.Rewriter = class Rewriter
       # Added support for spread dots on the left side: f ...a
       if (tag in IMPLICIT_FUNC and token.spaced or
           tag is '?' and i > 0 and not tokens[i - 1].spaced) and
-         (nextTag in IMPLICIT_CALL or nextTag is '...' or
+         (nextTag in IMPLICIT_CALL or 
+         (nextTag is '...' and @tag(i + 2) in IMPLICIT_CALL and not @findTagsBackwards(i, ['INDEX_START', '['])) or
```

Comment:
@helixbass tests don't fail because there are no many tests for spred dots on the left side.
`@tag(i +2)` is needed to catch cases like `{ a:1, b ... }` (note the space after the `b`).
In this case `tag` (b) is in `IMPLICIT_FUNC` and `nextTag` is `...`, so I need to check`@tag (i + 2)`. Otherwise, the `rewriter` will compile this into
`[IDENTIFIER b] [CALL_START (] [... ...] [CALL_END )]`.